tensorboard/webapp/metrics/views/card_renderer/scalar_card_linked_time_fob_test.ts

```diff
@@ -86,4 +86,41 @@ describe('ScalarLinkedTimeFobController', () => {
       );
     });
   });
+
+  describe('getHighestStep', () => {
+    it('gets the highest step when minMax is in order', () => {
+      const minMax: [number, number] = [0, 2];
+      const fixture = createComponent({minMax});
```

Comment:
minMax variable is not reused. this two lines could be combined.
const fixture = createComponent({minMax: [0, 2]});

and the same for all other test cases

Reply:
Makes sense to me. Changed.